core/eolearn/tests/test_core_tasks.py

```diff
@@ -11,16 +11,16 @@
 """
 
 import copy
-import datetime
 import pickle
+from datetime import datetime
 from typing import Dict, Iterable, Tuple, Union
 
 import numpy as np
 import pytest
 from fs.osfs import OSFS
 from fs.tempfs import TempFS
 from fs_s3fs import S3FS
-from numpy.testing import assert_equal
+from numpy.testing import assert_array_equal, assert_equal
```

Comment:
What is the difference between these two? I see both used in the same context below

Reply:
`assert_array_equal` expected inputs are np.array, `assert_equal` can take (scalars, lists, tuples, dictionaries or numpy arrays), `assert_array_equal` looks faster due to less checks constraints and numpy operations